README.md

```diff
@@ -11,6 +11,12 @@ dotnet restore
 dotnet tool restore
 ```
 
+- Formatter install/documentation
+```
+dotnet tool install -g dotnet-format
```

Comment:
Hi @gioacchinogioffre , thanks for checking this info!

Looking at the docs, it seems that dotnet 6+ already includes this tool so it wouldn't be necessary to install it: https://github.com/dotnet/format#how-to-use

> dotnet-format is now part of the .NET 6 SDK. Invoking the dotnet format command will fix whitespace, code style, and analyzer issues by default. dotnet-format will look in the current directory for a project or solution file and use that as the workspace to format.

That said, instead of this tool installation, I think we can add a new section `Formating` and include the instructions to run the formater:

Check format issues
```
dotnet format --verify-no-changes
```

Fix format issues
```
dotnet format
```